src/super_gradients/examples/resnet_qat/resnet_qat_example.py

```diff
@@ -0,0 +1,44 @@
+"""
+QAT example for Resnet18
+
+The purpose of this example is to demonstrate the usage of QAT in super_gradients through phase callbacks.
+"""
```

Comment:
what about QAT phase callback. 
I would like to be able to add QAT to a recipe, so that in epoch xx the QAT will start